### PR TITLE
Template spelling fixes

### PR DESCRIPTION
**Pull request by markknol**

### CODE REVIEW

File: 10-std.tex [hunk 1]
```diff
@@ -397,7 +397,7 @@ \section{Template}
 ::if isValid:: valid ::else:: invalid ::end::
 \end{lstlisting} 
 
-Operators can be used, but it doesn't deal with operator precedence. Therefore it is required to enclose each operation in parentheses \ic{()}. The current operators are allowed: \ic{+}, \ic{-}, \ic{*}, \ic{/}, \ic{>}, \ic{<},  \ic{>=}, \ic{<=}, \ic{==}, \ic{!=}, \ic{&&} and \ic{||}.
+Operators can be used but they doesn't deal with operator precedence. Therefore it is required to enclose each operation in parentheses \ic{()}. Currently, the following operators are allowed: \ic{+}, \ic{-}, \ic{*}, \ic{/}, \ic{>}, \ic{<},  \ic{>=}, \ic{<=}, \ic{==}, \ic{!=}, \ic{&&} and \ic{||}.
```
Comment (Simn) [inline 10-std.tex]: "they don't"

File: 10-std.tex [hunk 2]
```diff
@@ -442,14 +442,14 @@ \section{Template}
 The console will trace \ic{The users are Mark(30) John(45)}.
 
 \paragraph{Macros}
-To callback code while parts of the template are rendered, providing a \expr{macros} object to the argument of \expr{Template.execute}. The key will act the template variable name, the value refers to a callback function that should return a \type{String}. The first argument of this macro function is always a \expr{resolve()} method, followed by the given arguments. The resolve function can be called to retreive values from the Template context. If \expr{macros} has no such field, the result is unspecified.
+To callback code while parts of the template are being rendered, provide a \expr{macros} object to the argument of \expr{Template.execute}. The key will act as the template variable name, the value refers to a callback function that should return a \type{String}. The first argument of this macro function is always a \expr{resolve()} method, followed by the given arguments. The resolve function can be called to retrieve values from the template context. If \expr{macros} has no such field, the result is unspecified.
```
Comment (Simn) [inline 10-std.tex]: "callback" is not really a verb. "To call custom functions while..."

File: 10-std.tex [hunk 3]
```diff
@@ -442,14 +442,14 @@ \section{Template}
...
 
-The following example passes itself as macro function context, and executes \ic{display} from the template.
+The following example passes itself as macro function context and executes \ic{display} from the template.
 \haxe{assets/TemplateMacros.hx}
 The console will trace \ic{The results: Mark ran 3.5 kilometers in 15 minutes}.
 
 \paragraph{Globals}
-Use the \expr{Template.globals} object to store those values that should be applied across all Template instances. This has lower priority than the context argument of \expr{Template.execute}.
+Use the \expr{Template.globals} object to store the values that should be applied across all template instances. This has lower priority than the context argument of \expr{Template.execute}.
```
Comment (Simn) [inline 10-std.tex]: The upper-case Template was actually good here, but it should be haxe.Template to really get the point across.

Comment (Simn) [inline 10-std.tex]: Also I would lose the "the" that used to be "those" entirely. "object to store values that"

**Comment by markknol** [comment]
done

File: 10-std.tex [hunk 4]
```diff
@@ -441,15 +441,15 @@ \section{Template}
...
 
-The following example passes itself as macro function context, and executes \ic{display} from the template.
+The following example passes itself as macro function context and executes \ic{display} from the template.
 \haxe{assets/TemplateMacros.hx}
 The console will trace \ic{The results: Mark ran 3.5 kilometers in 15 minutes}.
 
 \paragraph{Globals}
-Use the \expr{Template.globals} object to store those values that should be applied across all Template instances. This has lower priority than the context argument of \expr{Template.execute}.
+Use the \expr{Template.globals} object to store values that should be applied across all \expr{haxe.Template} instances. This has lower priority than the context argument of \expr{Template.execute}.
```
Comment (Simn) [inline 10-std.tex]: nitpick: use \type for types so we can at some point link them to their API documentation.

Comment (markknol) [inline 10-std.tex]: Ok, should this still be a full path?

Comment (Simn) [inline 10-std.tex]: It should be \type{haxe.Template}

**Comment by Simn** [comment]
Thanks!

**Pull request merged**